package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "test": "jasmine",
     "lint": "eslint .",
-    "flow-check": "flow check"
+    "flow-check": "flow check",
+    "start": "node --max-old-space-size=4096 ./run-packager.js"    
   },
```

Comment:
Curious if this commit is intentional. I agree we should probably have some logical `npm start`, but can it go in a separate PR for discussion?

Reply:
Duh, was a mistake. I use this + 2 helper files to start the packager. That's because I've never been able to grok how to do that through existing commands, etc.
I'll remove. 

---
In reply to: [165935883](https://github.com/Microsoft/react-native-windows/pull/1616#discussion_r165935883) [](ancestors = 165935883)